charts/icm-as/values.yaml

```diff
@@ -67,10 +62,25 @@ jvm:
     # enable/disable to java-vm-debugging
     enabled: false
   # define options / parameters to be used to start the jvm
-  options: "-XX:MaxRAMPercentage=80 -XX:InitialRAMPercentage=80"
+  options: "-XX:NewRatio=8 -XX:MaxRAMPercentage=80 -XX:InitialRAMPercentage=80"
   # define additional options / parameters to be used to start the jvm (will be appended to options)
   additionalOptions:
 
+# configuration related to Datadog
+datadog:
+  # enable or disable the Datadog Java agent to monitor the icm-as JVM
+  enabled: false
+  # define the environment (dd.env)
+  env: "tenant-develop"
+  # override service name (dd.service) (default: icm-as)
+  # service: "icm-as"
+  # override version (dd.version) (default: container version of icm-as)
+  # version: "2022.12.24"
+  # enable profiling  (dd.profiling.enabled, dd.trace.analytics.enabled, dd.jdbc.analytics.enabled
```

Comment:
Please adapt the comment to reflect the actual meaning

Reply:
missing bracket in the end of the line
@skiesewetter-intershop: Could do a proposal for the "meaning" text?